### PR TITLE
Package kubectl 1.4.7

### PR DESCRIPTION
**Pull request by itskingori**
Title says it all.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM alpine:3.5
+
+MAINTAINER Zappi DevOps <devops@zappistore.com>
+
+ARG APP_DEPS="ca-certificates"
+ARG BUILD_DEPS="wget"
+ARG KUBECTL_VERSION="v1.4.7"
+ARG KUBECTL_PACKAGE="kubernetes-client-linux-amd64.tar.gz"
+ARG KUBECTL_RBENV_SHA="36232c9e21298f5f53dbf4851520a8cc53a2d6b6d2be8810cf5258a067570314"
```
Comment (adrianmoisey) [inline Dockerfile]: This ARG seems strangely named. 
What about `KUBECTL_SHA` ?

Comment (itskingori) [reply Dockerfile]: LOL. Copypasta ... 🙈  ... fixed in 98c37a8.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+RUN apk update && \
+    apk upgrade && \
+    apk add $APP_DEPS && \
+    apk add $BUILD_DEPS && \
+    wget --progress=dot:mega https://dl.k8s.io/${KUBECTL_VERSION}/${KUBECTL_PACKAGE} && \
+    echo "${KUBECTL_SHA} *${KUBECTL_PACKAGE}" | sha256sum -c - && \
+    tar --no-same-owner -xzf ${KUBECTL_PACKAGE} && \
+    mv /kubernetes/client/bin/kubectl /usr/local/bin/kubectl && \
+    rm -R /kubernetes* && \
```
Comment (adrianmoisey) [inline Dockerfile]: This `rm` is unneeded, as there is already an `rm` on line 21.

Comment (itskingori) [reply Dockerfile]: Good catch. Fixed in e23c749.

**Review by adrianmoisey — APPROVED** [review]

**Comment by adrianmoisey** [comment]
:+1:

**Pull request merged**